src/cli/commands/force/lightning/local/device/__tests__/create.test.ts

```diff
@@ -97,21 +98,15 @@ describe('List Tests', () => {
             createNewDeviceMock
         );
 
-        await create.run();
+        create.validateInputParameters();
+        await create.executeDeviceCreate();
         expect(createNewDeviceMock).toHaveBeenCalledWith(
             deviceName,
             iOSDeviceType,
             iOSSupportedRuntimes[0]
         );
     });
 
-    test('Logger must be initialized and invoked', async () => {
```

Comment:
There's no value in this test. `create` command extends `setup` and we already test this in setup.